src/components/AgenciesPage/components/components/components/AgenciesItemContent/AgenciesItemContent.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { CardContent, Typography } from '@mui/material';
+import { SpecificationsTable } from './components';
+import type { AgencyItemContent } from '../../../../types/Agencies';
+import './AgenciesItemContent.scss';
+
+interface AgenciesItemContentProps {
+  agency: AgencyItemContent;
+}
+
+export const AgenciesItemContent: React.FC<AgenciesItemContentProps> = ({
+  agency,
+}) => {
+  return (
+    <CardContent className="agencies-item-card">
+      <Typography
+        className="agencies-item-card-content"
+        gutterBottom
+        variant="h5"
+        component="div"
+      >
+        {agency.name}
+      </Typography>
+
+      <div className="card-details">
```

Comment:
Why do you need `div` with `className`?

Reply:
because I need flex-container in which the table is.

Reply:
I have not found styles for `card-details`

Reply:
sory|
looked at other class 
moved this `div`